api/src/resources/comment.rs

```diff
@@ -120,6 +120,9 @@ pub struct Comment {
     #[serde(skip_serializing_if = "PropertyMap::is_empty", default)]
     pub user_properties: PropertyMap,
     pub created_at: DateTime<Utc>,
+
+    #[serde(default = "default_is_false")]
```

Comment:
why not `#[serde(default)]`, the `Default` impl is `false` for `bool` so i think this should work